src/pages/news-announcements.tsx

```diff
@@ -19,13 +20,15 @@ import styles from 'styles/pages/news.module.scss'
 import type { RSSNewsItem } from 'types'
 import { validateNewsItems, formatToArticleListItem } from 'helpers/index'
 import { GET_ANNOUNCEMENTS } from 'operations/cms/queries/getAnnouncements'
+import { GET_INTERNAL_NEWS_CAROUSEL_ARTICLES } from 'operations/cms/queries/getInternalNewsCarouselArticles'
 import { SPACEFORCE_NEWS_RSS_URL } from 'constants/index'
 import { ArticleList } from 'components/ArticleList/ArticleList'
 
 const RSS_URL = `${SPACEFORCE_NEWS_RSS_URL}&max=12`
```

Comment:
If we only want to display 4 on the page and don't need the rest, we can change `&max=12` to `&max=4`

Reply:
Awesome! I was looking for a solution like this but completely passed this line over. Thanks!